assets/src/edit-story/utils/isDefaultPage.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createPage } from '../elements';
+
+/**
+ * @typedef {import('../../../types').Page} Page
+ */
```

Comment:
Let's expand here the use of this function as well—like `isDefaultPage`. I assume this is just like reverse shallowEqual.

Reply:
I had mean't to ask if there was already one in the codebase because I couldn't find one at the time, but I just did! Replaced with one from `react-pure-render`.